man/froll.Rd

```diff
@@ -83,37 +69,68 @@ frollapply(x, n, FUN, \dots, fill=NA, align=c("right", "left", "center"))
   to compute a rolling average is inefficient. It is also always single-threaded
   because there is no thread-safe API to R's C \code{eval}. Nevertheless we've
   seen the computation speed up vis-a-vis versions implemented in base R.
+
+  Setting \code{options(datatable.verbose=TRUE)} will display various
+  information about how rolling function processed. It will not print
+  information in a real-time but only at the end of the processing.
```

Comment:
```suggestion
  Setting \code{options(datatable.verbose=TRUE)} will display various
  information about how rolling function processed. It will not print
  information in real-time but only at the end of the processing.
```